core/common/src/main/java/alluxio/network/protocol/RPCProtoMessage.java

```diff
@@ -126,7 +126,7 @@ public void encode(ByteBuf out) {
    * @param prototype a message prototype used to infer the type of the message
    * @return the message decoded
    */
-  public static RPCProtoMessage decode(ByteBuf in, ProtoMessage.Type prototype) {
+  public static RPCProtoMessage decode(ByteBuf in, ProtoMessage prototype) {
```

Comment:
i was thinking "prototype" as one word which means a 'prototype' of the message to return.  I borrowed the naming convention from netty (https://netty.io/4.0/api/io/netty/handler/codec/protobuf/ProtobufDecoder.html#ProtobufDecoder-com.google.protobuf.MessageLite-)